src/components/data-processing/CellSizeDistribution/CellSizeDistribution.jsx

```diff
@@ -94,6 +97,16 @@ const CellSizeDistribution = (props) => {
     (state) => state.componentConfig[plots[selectedPlot].plotUuid]?.plotData,
   );
 
+  const histogramPlotData = useSelector(
+    (state) => state.componentConfig[plots.histogram.plotUuid]?.plotData,
+  );
+
+  const highestUmiRef = useRef(0);
```

Comment:
I am unfamiliar with `useRef`. Is there a reason that makes it a better choice than `useState()` for this value?

Reply:
they are used for different things: useRef is a way to persist the value inside across components re-renders, so keep it across the full lifetime of the component. Since we know that this value doesn't need to be recalculated whenever we re-render I thought it would be a valid use of useRef.

useState is for when we want to have a value that will affect what gets rendered in the component, but in this case it isn't affecting it but rather its children (we are passing it as props).

Reply:
Thanks! Very useful!